src/drivers/amd/vangogh/acp_bt_dma.c

```diff
@@ -0,0 +1,430 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+//Copyright(c) 2023 AMD. All rights reserved.
+//
+//Author:	Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+//		SaiSurya, Ch <saisurya.chakkaveeravenkatanaga@amd.com>
+
+#include <rtos/atomic.h>
+#include <sof/audio/component.h>
+#include <rtos/bit.h>
+#include <sof/drivers/acp_dai_dma.h>
+#include <rtos/interrupt.h>
+#include <rtos/timer.h>
+#include <rtos/alloc.h>
+#include <rtos/clk.h>
+#include <sof/lib/cpu.h>
+#include <sof/lib/dma.h>
+#include <sof/lib/io.h>
+#include <sof/lib/pm_runtime.h>
+#include <sof/lib/notifier.h>
+#include <sof/platform.h>
+#include <sof/schedule/schedule.h>
+#include <rtos/spinlock.h>
+#include <sof/math/numbers.h>
+#include <sof/trace/trace.h>
+#include <ipc/topology.h>
```

Comment:
that's a colourful collection... topology.h? schedule.h? For every function or type or object there should be a header included (except where a forward declaration is enough), that's correct. But unneeded headers should be avoided

Reply:
Will take care in future commits.